packages/babel-preset-default/bin/index.js

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env node
+
+require( 'core-js-builder' )( {
+	modules: [ 'es', 'web' ],
+	exclude: [ 'es.promise' ],
+	targets: require( '@wordpress/browserslist-config' ),
```

Comment:
One way to test it is to set some random settings like:
```js
targets: '> 90%',
```

and check the size of the generated `polyfill.js` file.

Reply:
Confirmed that it generates different output based on the config 👍